stringer.go

```diff
@@ -434,35 +472,35 @@ func (f *File) genDecl(node ast.Node) bool {
 		// We now have a list of names (from one line of source code) all being
 		// declared with the desired type.
 		// Grab their names and actual values and store them in f.values.
-		for _, name := range vspec.Names {
-			if name.Name == "_" {
+		for _, n := range vspec.Names {
```

Comment:
The scope of the variable `n` is sufficiently large to deserve a longer name. I would stick with the previous name `name`. What was wrong with it?